task-maker-format/Cargo.toml

```diff
@@ -61,5 +61,3 @@ wildmatch = "2.1.0"
 [dev-dependencies]
 tempfile = "3.3"
 pretty_assertions = "1.2"
-# assert_that! macro
-spectral = "0.6"
```

Comment:
It's sad that `spectral` is dead (pun not intended). However there was a point with using it: when the assertion fails (sometimes) the output is much nicer out of the box. For example `assert!(X.contains(Y))` does not print X nor Y, just "assertion failed". This may be quite annoying in the CI for example (which ironically just happened with the first version of this PR).

Why do you want to remove it instead of replacing it with a maintained fork? (e.g. https://crates.io/crates/speculoos)

Reply:
Migrated to speculoos